### PR TITLE
fix(sidebar): fix sidebar by new method

### PR DESCRIPTION
**Pull request by sapirmosko**
fix(sidebar): fix sidebar by new method

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (amnon3234) [inline .vscode/settings.json:2]: please add the .vscode to gitignore

File: clients/dashboard-service/src/index.css [hunk 2]
```diff
@@ -13,11 +13,72 @@
...
+}
+
+ul ul a {
+  font-size: 0.9em !important;
+  padding-left: 30px !important;
+  background: #EB7A02;
+}
+
+a, a:hover, a:focus {
+  color: inherit;
```
Comment (amnon3234) [inline clients/dashboard-service/src/index.css:81]: cursor: pointer; and change the color on hover

File: .vscode/settings.json (hunk 1 above)
Comment (sapirmosko) [reply .vscode/settings.json]: fixed

File: clients/dashboard-service/src/index.css (hunk 2 above)
Comment (sapirmosko) [reply clients/dashboard-service/src/index.css]: fixed

**Pull request merged**